sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Data_Inventory_Report',
+        'select
+            -- Personal Info
+            users.first_name as "First Name",
+            users.last_name as "Last Name",
+            students.birth_date as "Date of Birth",
+            students.gender as "Gender",
+            sin_validations.sin as "SIN",
+            users.email as "Email address",
+            students.contact_info -> ''address'' ->> ''addressLine1'' as "Address Line 1",
+            students.contact_info -> ''address'' ->> ''addressLine2'' as "Address Line 2",
+            students.contact_info -> ''address'' ->> ''city'' as "City",
```

Comment:
The requirement file says Contact city
![image](https://github.com/bcgov/SIMS/assets/54600590/502c783d-db0e-4ac5-8289-f2affedb9910)

Likewise for province.

Reply:
I can change that but I think that all the titles should be revised by the business for them to be clearer. Some of them are duplicated as in the parents sections.

Reply:
It will be easy for us to simply follow the document and business can update later may be during the upcoming ticket.